### PR TITLE
Release/v3.4.0 v1

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

v3.4.0 RC 1

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Comment by sethkfman** [comment]
@Cal-L Is there a dependency update for contract-metadata?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 
-## Current Develop Branch
-- Update build versions
+## v3.4.0 - September 25 2021
+- [#3171](https://github.com/MetaMask/metamask-mobile/pull/3171): [FIX] Update contract-metadata library to v1.30.0
```
Comment (gantunesr) [inline CHANGELOG.md:4]: Do you think this is relevant to the user?

**Review by gantunesr — APPROVED** [review]
lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (gantunesr) [reply CHANGELOG.md:4]: never mind!

Comment (Cal-L) [reply CHANGELOG.md:4]: I think change logs are more for devs but yeah library upgrades could be informational.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 
-## Current Develop Branch
-- Update build versions
+## v3.4.0 - September 25 2021
```
Comment (sethkfman) [inline CHANGELOG.md:3]: Make sure to keep the `## Current Develop Branch` in the document

**Pull request merged**